app/models/colony.rb

```diff
@@ -7,4 +7,6 @@ class Colony < ActiveRecord::Base
 
   validates :name, :street_address, :city, :state, presence: true
   validates :pop, numericality: true, allow_nil: true
+  validates :lat, :lng, numericality: true
```

Comment:
this may be why the create colony tests are failing - they don't set the lat and lng attributes, so either we need to allow nil or add these attributes to those tests (and be sure that they can be added when a user creates a colony)